packages/combo-box/test/external-filtering.test.js

```diff
@@ -27,7 +27,7 @@ describe('external filtering', () => {
       expect(comboBox._scroller.items).to.eql(['foo', 'bar', 'baz']);
     });
 
-    it('should remove focus while loading', async () => {
+    it.skip('should remove focus while loading', async () => {
```

Comment:
Is skipping here intentional as a consequence of this change? The test passes locally for me.